Library/brew.rb

```diff
@@ -68,6 +68,7 @@ def require? path
   empty_argv = ARGV.empty?
   help_regex = /(-h$|--help$|--usage$|-\?$|^help$)/
   help_flag = false
+  internal_cmd = true
```

Comment:
@mikemcquaid I just added this line, because without it any exception raised between the beginning of the block and the `internal_cmd` assignment wouldn’t trigger the “please report this bug” message. Does it makes sense?